source/docs/software/basic-programming/git-getting-started.rst

```diff
@@ -0,0 +1,260 @@
+Git Version Control Introduction
+================================
+
+`Git <https://git-scm.com/about>`_ is a Distributed Version Control System (VCS) created by Linus Torvalds, also known for creating and maintaining the linux kernel. Version Control is a system for tracking changes of code for developers. The advantages of Git Version Control are:
+
+- Seperate testing environments into *branches*
+- Ability to navigate to a particular *commit* without removing history
+- Ability to manage *commits* in various ways, including combining them
+- Various other features, see `here <https://git-scm.com/about>`__
+
+Prerequisites
+-------------
+
+You have to download and install Git from the following links:
+
+- `Windows <https://git-scm.com/download/win>`_
+- `Mac OS X <https://git-scm.com/download/mac>`_
+- `Linux <https://git-scm.com/download/linux>`_
+
+.. note:: You may need to add Git to your `path <https://www.google.com/search?q=adding+git+to+path>`__
+
+Git Vocabulary
+--------------
+
+Git is composed of several primary elements:
+
+- **Repository:** the data structure of your code, including a ``.git`` folder in the root directory
+- **Commits:** a particular saved state of the repository, this includes all files and additions
+- **Branches:** a means of separating various commits, having a unique history. This is primarily used for separating development and stable branches.
+- **Push:** update the remote repository with your local changes
+- **Pull:** update your local repository with the remote changes
+- **Cloning:** retrieving a local copy of a repository to modify
```

Comment:
Inconsistent parallel structure; some of these are nouns, some of them are the actual command name, some of them are gerunds.  Make this consistent.